sklearn/ensemble/_hist_gradient_boosting/loss.py

```diff
@@ -141,6 +153,55 @@ def update_gradients_and_hessians(self, gradients, hessians, y_true,
         _update_gradients_least_squares(gradients, y_true, raw_predictions)
 
 
+class LeastAbsoluteDeviation(BaseLoss):
+    """Least asbolute deviation, for regression.
+
+    For a given sample x_i, the loss is defined as::
+
+        loss(x_i) = |y_true_i - raw_pred_i|
+    """
+
+    hessians_are_constant = True
+    need_update_leaves_values = True  # see note in BaseLoss
```

Comment:
Maybe just a note in the docstring would be better avoiding to go to see the `BaseLoss`. It can be in the 2 places, I don't mind.